src/modules/hip/kernel/glitch.hpp

```diff
@@ -0,0 +1,278 @@
+#include <hip/hip_runtime.h>
+#include "rpp_hip_common.hpp"
+
+template <typename T>
+__device__ __forceinline__ void rpp_hip_load1_glitch(T *srcPtr, uint2 srcStrideCH, float locSrcX, float locSrcY, float *dst, int channels)
```

Comment:
Ideally call by reference here as - float &locSrcX, float &locSrcY, float &dst

Reply:
We don't need call by reference for locSrcX and locSrcY as we are not using those after this code. However, I am calling this function like this, where dst is passed by reference:
`rpp_hip_load1_glitch(srcPtr, srcStrideCH, srcX_f8->f1[0], srcY_f8->f1[0], &(dst_f8->f1[0]), channels);`

Reply:
Done